src/main/java/com/icthh/xm/tmf/ms/communication/channel/telegram/TelegramChannelHandler.java

```diff
@@ -0,0 +1,123 @@
+package com.icthh.xm.tmf.ms.communication.channel.telegram;
+
+import static com.icthh.xm.tmf.ms.communication.utils.LogUtil.withLog;
```

Comment:
probably is worth to move this utility to xm-commons-logging

Reply:
+

Reply:
no, because in this case we will not see class, which wrote this log message